src/Illuminate/Container/Container.php

```diff
@@ -602,6 +602,8 @@ public function makeWith($abstract, array $parameters = [])
      * @param  string  $abstract
      * @param  array  $parameters
      * @return mixed
+     *
+     * @throws \Illuminate\Contracts\Container\BindingResolutionException
```

Comment:
@xuanquynh This should be added to the Container interface docblock too, right? :)

Reply:
@X-Coder264 I agree with you but it's a breaking change. I am not sure Mr.Otwell want to add it to the Container interface. Someone may do that.

Reply:
@xuanquynh From a purely theoretical standpoint, yes. But realistically looking the exception in question is in the contracts namespace which means that at least some method from the contract is supposed to throw it (otherwise it wouldn't exist there in the first place). Besides you did send this PR to the master (5.8) branch which is precisely the branch where you send breaking changes so I don't see a problem :)

Reply:
@X-Coder264 Ok. I will create a new pull request soon and reference it with this. :)